projects/pastanaga-angular/src/lib/controls/base-control.ts

```diff
@@ -394,9 +394,7 @@ export abstract class BaseControl implements OnChanges, OnInit, OnDestroy, Contr
             this.control.setAsyncValidators(this._parentAsyncValidator);
         }
 
-        if (this.control.dirty) {
-            this.control.updateValueAndValidity();
-        }
+        this.control.updateValueAndValidity();
```

Comment:
in angular component, this behavior can be toggled using an injected config
I think we should do the same
because the behavior that you are changing was definied by UX in order not to display errors when a field is pristine

Reply:
I don't think I'm aware of the config you are referencing.  The issue I am trying to address is that form is not being marked as invalid as it should.

As an aside, I question why any validators created externally are always replaced by internal validators.

Reply:
in angular components, it's accomplished using errorStateMatcher

indeed, on a pristine state we won't evaluate the validity of the fields... which was an ux decision.
pristine being a state where you cannot submit anyway.

to give you more insight of this internal validation thing... external validators are not "replaced", there are eventually completed with "custom" validators defined using some @input of pastanaga form fields.
this was the spec... 

if you want to evalaute the valid status of your form before the user change a value in it, you can always mark it as dirty...